packages/features/ee/workflows/components/TimeTimeUnitInput.tsx

```diff
@@ -39,14 +40,14 @@ export const TimeTimeUnitInput = (props: Props) => {
           className="-mt-2 rounded-r-none text-sm focus:ring-0"
           {...form.register("time", { valueAsNumber: true })}
           addOnSuffix={
-            <Dropdown>
+            <Dropdown onOpenChange={setIsDropdownOpen}>
               <DropdownMenuTrigger asChild>
                 <button className="flex items-center">
-                  <div className="mr-1 w-3/4">
+                  <div className="mr-1 w-3/5">
                     {timeUnit ? t(`${timeUnit.toLowerCase()}_timeUnit`) : "undefined"}{" "}
                   </div>
-                  <div className="w-1/4 pt-1">
```

Comment:
@Sahillather002 you can keep the width here as `w=1/4`

Reply:
@Ryukemeister i did it because icon was hiding to that width so gives larger width, or i can reduce the size of icon